tests/raft_node.py

```diff
@@ -95,40 +95,9 @@ async def _unittest_raft_node_init() -> None:
     assert len(raft_node._request_vote_clients) == 1
     assert len(raft_node._append_entries_clients) == 1
     assert raft_node._next_index == [1]
-    # assert raft_node._match_index == [0]
-
-
-async def _unittest_raft_node_term_timeout() -> None:
-    """
-    Test that the LEADER node term is increased upon term timeout
-
-    Test that the CANDIDATE/FOLLOWER node term is not increased upon term timeout
-    """
-
-    os.environ["UAVCAN__NODE__ID"] = "41"
-    raft_node = RaftNode()
-    ELECTION_TIMEOUT = 10  # so that we don't start an election
-    TERM_TIMEOUT = 1
-    raft_node.election_timeout = ELECTION_TIMEOUT
-    raft_node.term_timeout = TERM_TIMEOUT
-
-    asyncio.create_task(raft_node.run())
-    raft_node._change_state(RaftState.CANDIDATE)
-    raft_node._change_state(RaftState.LEADER)  # only leader can increase term
-
-    await asyncio.sleep(TERM_TIMEOUT)  # + 0.1 to make sure the timer has been reset
-    assert raft_node._term == 1
-    await asyncio.sleep(TERM_TIMEOUT)
-    assert raft_node._term == 2
-    await asyncio.sleep(TERM_TIMEOUT)
-    assert raft_node._term == 3
-
-    raft_node._change_state(RaftState.FOLLOWER)  # follower should not increase term
-    await asyncio.sleep(TERM_TIMEOUT)
-    assert raft_node._term == 3
```

Comment:
Removing is not necessary, adjust it so it tests that once the Leader node is elected the term is not increased, regardless of how many term timeouts would happen.